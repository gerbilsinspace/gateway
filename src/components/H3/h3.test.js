import React from 'react';
import { shallow } from 'enzyme';
import H3 from './index';

let component;

describe('H3', () => {
  beforeEach(() => {
    component = shallow(<H3 />);
  });

  it('should render without crashing', () => {
    expect(component).toBeDefined();
  });

  it('should match snapshot with no data', () => {
    expect(component).toMatchSnapshot();
  });
})