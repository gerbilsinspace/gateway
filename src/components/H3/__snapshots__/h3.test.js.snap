// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H3 should match snapshot with no data 1`] = `
<h3
  className="sc-bdVaJa bDWFJH"
/>
`;
